scopes/lanes/lanes/merge-lanes.ts

```diff
@@ -37,7 +40,7 @@ export async function mergeLanes({
   const localLane = currentLaneId.isDefault() ? null : await consumer.scope.loadLane(localLaneId);
   const laneId = new LaneId({ name: laneName });
   let bitIds: BitId[];
-  let otherLane: Lane | null;
+  let otherLane: Lane | null | undefined;
```

Comment:
why do you need null here?

Reply:
it could be null because
`await consumer.scope.loadLane(laneId);`
returns a Promise<Lane | null>

It is null when you try to query the main lane or an invalid lane name.

Reply:
why not return undefined from loadLane?